### PR TITLE
implemented cart management

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
Implemented cart management
- add to cart
- quantity increment and decrement
- total price calculation
- remove from cart

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -5,4 +5,7 @@
 
 .margin-top {
   margin: 1rem 0rem;
```
Comment (anshtripathi01) [inline src/App.css:7]: Here you can remove rem it's still worked.

File: src/components/navbar.jsx [hunk 2]
```diff
@@ -1,8 +1,10 @@
+import { Link } from "react-router-dom";
+
 export const Navbar = () => {
     return(
         <nav className="nav space-around">
           <div> 
-              <a className="nav-header" href="/">The Sports Adda</a>
```
Comment (anshtripathi01) [inline src/components/navbar.jsx:5]: You can replace anchor tag with link or button

File: src/components/navbar.jsx [hunk 3]
```diff
@@ -11,9 +13,9 @@ export const Navbar = () => {
              <a href="./Authentication/login.html">
               <button className="btn solid-btn submit">Login</button>  
           </a>
-              <a className="nav-item" href="./CartManagement/cartManagement.html">
```
Comment (anshtripathi01) [inline src/components/navbar.jsx:14]: Keep above points here also

**Review by anshtripathi01 — APPROVED** [review]
Code,format and Ui looks good. Go ahead

File: src/App.css (hunk 1 above)
Comment (govardhanreddyc79) [reply src/App.css:7]: okay

File: src/components/navbar.jsx (hunk 2 above)
Comment (govardhanreddyc79) [reply src/components/navbar.jsx:5]: done that

File: src/components/navbar.jsx (hunk 3 above)
Comment (govardhanreddyc79) [reply src/components/navbar.jsx:14]: Removed already

File: src/components/cartProductCard.jsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import { useCart } from "context/cart-context";
+export const CartProductCard = ({cartProduct}) => {
+    console.log(cartProduct.quantity)
```
Comment (amirnathani9) [inline src/components/cartProductCard.jsx:3]: remove logs

**Review by amirnathani9 — APPROVED** [review]
clean code practices followed
good folder structure
good to merge

**Pull request merged**